add library panels rbac (#76483)
grafana/grafana: docs/sources/whatsnew/whats-new-next/index.md

The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.

To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).

## RBAC for library panels

<!-- Kay Delaney, Natalia Bernarte  -->

> <!-- Enterprise, Cloud -->
> 
> _Available in public preview in Grafana Enterprise and Generally available in Grafana Cloud_

Reviewer: I'll add the specifics of the Enterprise availability when I copy this over to the on-prem What's new. Also, we want to add a date in the Cloud version of this; is it just the 10.2 release date?
```suggestion
<!-- Enterprise public preview -->

<Date>

_Generally available in Grafana Cloud_
```

Author: It's just 10.2

> You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/) which provides a way to extend the permissions avaiable with basic roles and enables you more granular control of users' actions.

Reviewer: ```suggestion
You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/), which provides a way to extend the permissions available with basic roles and gives you more granular control of users' actions.
```

The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.

To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).

## RBAC for library panels

<!-- Kay Delaney, Natalia Bernarte  -->
<!-- Enterprise public preview -->

> <Date>

Reviewer: ```suggestion
October 24, 2023
```

_Generally available in Grafana Cloud_

> You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/), which provides a way to extend the permissions available with basic roles and gives you more granular control of users' actions.

Reviewer: ```suggestion
You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/access-control/), which provides a way to extend the permissions available with basic roles and gives you more granular control of users' actions.
```
